ert3/evaluator/_evaluator.py

```diff
@@ -185,17 +206,23 @@ def command_location(name: str) -> FilePath:
         }
     ]
 
+    storage_conf = {
+        "type": ensemble.storage_type,
+        "storage_path": evaluation_tmp_dir / ".my_storage",
+    }
+
+    if ensemble.storage_type == "ert_storage":
+        workspace = evaluation_tmp_dir.parent.parent.parent
+        storage_conf["storage_path"] = ert.storage.get_records_url(workspace)
+
     ee_config: Dict[str, Any] = {
         "steps": steps,
         "realizations": ensemble_size,
         "max_running": 10000,
         "max_retries": 0,
         "run_path": evaluation_tmp_dir / "my_output",
         "executor": ensemble.forward_model.driver,
-        "storage": {
-            "type": "shared_disk",
-            "storage_path": evaluation_tmp_dir / ".my_storage",
-        },
+        "storage": storage_conf,
```

Comment:
Could we try to get the `storage` part removed from the `ee_config`? Looking at the prefect ensemble, it appears that this info is not used (which I think is the way it should be), so maybe we could pass this info directly to the functions in this module that needs it instead, so that it is kept local?

Reply:
I also think the `storage_path` is getting slightly overloaded, but it would help a lot if we did this :arrow_up:, so that it's not an implicit part of the interface to this module.

Reply:
good point I will try to remove it from the ee_config